plot_types/basic/stackplot.py

```diff
@@ -0,0 +1,27 @@
+"""
+===============
+stackplot(x, y)
+===============
+See `~matplotlib.axes.Axes.stackplot`
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+plt.style.use('_mpl-gallery')
+
+# make data
+x = np.arange(0, 10, 2)
+ay = [1, 1.25, 2, 2.75, 3]
+by = [1, 1, 1, 1, 1]
+cy = [2, 1, 2, 1, 2]
+y = np.vstack([ay, by, cy])
+labels = ['a', 'b', 'c']
```

Comment:
```suggestion

```
Not used anymore. And we have an empty line before the plot comment in all the examples.